recipes/flyr/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "flyr" %}
+{% set version = "4.0.0" %}
+{% set python_min = "3.9" %}
```

Comment:
```suggestion
```
This is not needed because the default value for `python_min` is already 3.9.

Reply:
Thanks @danielnachun, we removed this